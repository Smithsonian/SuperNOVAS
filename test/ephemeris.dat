
2450545.000000 SUN        S0 O0 A1:    -0.006750     0.004995     0.002339    -0.000006    -0.000006    -0.000002     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2450545.000000 SUN        S0 O1 A1:    -0.006750     0.004995     0.002339    -0.000006    -0.000006    -0.000002     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2450545.000000 SUN        S0 O2 A1:    -0.006750     0.004995     0.002339    -0.000006    -0.000006    -0.000002     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2451545.000000 SUN        S0 O0 A1:    -0.007138    -0.002644    -0.000921     0.000005    -0.000007    -0.000003     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2451545.000000 SUN        S0 O1 A1:    -0.007138    -0.002644    -0.000921     0.000005    -0.000007    -0.000003     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2451545.000000 SUN        S0 O2 A1:    -0.007138    -0.002644    -0.000921     0.000005    -0.000007    -0.000003     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2452545.000000 SUN        S0 O0 A1:     0.000657    -0.004704    -0.002013     0.000008     0.000003     0.000001     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2452545.000000 SUN        S0 O1 A1:     0.000657    -0.004704    -0.002013     0.000008     0.000003     0.000001     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 

2452545.000000 SUN        S0 O2 A1:     0.000657    -0.004704    -0.002013     0.000008     0.000003     0.000001     0.000000     0.000000     0.000000     0.000000     0.000000     0.000000 
