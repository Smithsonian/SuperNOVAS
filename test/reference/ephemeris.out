  -10000.000 A0: ERROR 13 
  -10000.000 A0: ERROR 13 
  -10000.000 A0: ERROR 13 
  -10000.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
       0.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.000 A0: ERROR 13 
   10000.010 A0: ERROR 13 
   10000.010 A0: ERROR 13 
   10000.010 A0: ERROR 13 
   10000.010 A0: ERROR 13 
  -10000.000 A1: SUN        0   -9.16286e-06    0.003772903     0.00165138   -16.877    -4.412    -1.552 

  -10000.000 A1: SUN        1              0              0              0     0.000     0.000     0.000 

  -10000.000 A1: EARTH      0      0.8100584     -0.5537621     -0.2401058 30082.393 37688.408 16343.251 

  -10000.000 A1: EARTH      1      0.8100676      -0.557535     -0.2417572 30099.270 37692.820 16344.803 

       0.000 A1: SUN        0   -0.007137516   -0.002643738  -0.0009213423    15.988   -20.395    -9.153 

       0.000 A1: SUN        1              0              0              0     0.000     0.000     0.000 

       0.000 A1: EARTH      0     -0.1842713      0.8847817      0.3838245 -51571.356 -8707.491 -3775.488 

       0.000 A1: EARTH      1     -0.1771338      0.8874255      0.3847458 -51587.344 -8687.096 -3766.334 

   10000.000 A1: SUN        0  -9.931662e-06   -0.004017579   -0.001668776    13.583    11.479     4.632 

   10000.000 A1: SUN        1              0              0              0     0.000     0.000     0.000 

   10000.000 A1: EARTH      0     -0.5372355      -0.790493     -0.3425861 42862.761 -25283.187 -10960.321 

   10000.000 A1: EARTH      1     -0.5372256     -0.7864754     -0.3409173 42849.178 -25294.666 -10964.953 

   10000.000 A1: SUN        0  -9.931662e-06   -0.004017579   -0.001668776    13.583    11.479     4.632 

   10000.000 A1: SUN        1              0              0              0     0.000     0.000     0.000 

   10000.000 A1: EARTH      0     -0.5372355      -0.790493     -0.3425861 42862.761 -25283.187 -10960.321 

   10000.000 A1: EARTH      1     -0.5372256     -0.7864754     -0.3409173 42849.178 -25294.666 -10964.953 

   10000.010 A1: SUN        0  -9.886359e-06   -0.004017541   -0.001668761    13.582    11.479     4.632 

   10000.010 A1: SUN        1              0              0              0     0.000     0.000     0.000 

   10000.010 A1: EARTH      0     -0.5370925     -0.7905773     -0.3426226 42867.327 -25276.479 -10957.412 

   10000.010 A1: EARTH      1     -0.5370826     -0.7865597     -0.3409538 42853.745 -25287.958 -10962.045 

